src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -78,6 +79,107 @@ def get_dsn_content(self, dataset_name):
         response_json = self.request_handler.perform_request("GET", custom_args)
         return response_json
 
+    def create_data_set(self, dataset_name, options = 
+        {
+            "volser": "zmf046",
+            "unit": "3390",
+            "dsorg": "PS",
+            "alcunit": "TRK",
+            "primary": 10,
+            "secondary": 5,
+            "dirblk": 10,
+            "avgblk": 500,
+            "recfm": "FB",
+            "blksize": 400,
+            "lrecl": 80,
+            "dsntype":"LIBRARY"
+        }):
+
+        """
+        Create a sequential or partitioned dataset.
+        Parameters
+        ----------
+            dataset_name
+        Returns
+        -------
+        json
+        """
+        
+        for opt in options:
+            if opt == "volser":
+                if options[opt] == None:
+                    options[opt] == "zmf046"
+                elif options[opt] != "zmf046":
+                    raise KeyError
+            if opt == "unit":
+                if options[opt] == None:
+                    options[opt] = "3390"
+            if opt == "dsorg":
+                if options[opt] == None:
+                    options[opt] = "PS"
+                else:
+                    if options[opt] not in ("PO", "PS"):
+                        raise KeyError
+            if opt == "alcunit":
+                if options[opt] == None:
+                    options[opt] = "TRK"
+                else:
+                    if options[opt] not in ("CYL", "TRK"):
+                        raise KeyError
+            if opt == "primary":
+                if options[opt] == None:
+                    options[opt] = 10
+            if opt == "secondary":
+                if options[opt] == None:
+                    options[opt] = 5
+            if opt == "dirblk":
+                if options[opt] == None:
+                    options[opt] = 10
```

Comment:
Some validation we could do for "dirblk" is:
* If `options["dsorg"] == "PS"`, then `dirblk` should be zero.
* If `options["dsorg"] == "PO"`, then `dirblk` should be non-zero.

The reason for this is that PS (sequential) datasets don't have directory blocks, but PO datasets do because they contain multiple members (similar to how a directory can contain multiple files).